src/pages/teachers/specialist/programmes/[programmeSlug]/units.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import {
+  GetStaticPathsResult,
+  GetStaticProps,
+  GetStaticPropsResult,
+  NextPage,
+} from "next";
+
+import {
+  getFallbackBlockingConfig,
+  shouldSkipInitialBuild,
+} from "@/node-lib/isr";
+import AppLayout from "@/components/SharedComponents/AppLayout";
+import { getSeoProps } from "@/browser-lib/seo/getSeoProps";
+import getPageProps from "@/node-lib/getPageProps";
+import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
+import { SpecialistUnitListingData } from "@/node-lib/curriculum-api-2023/queries/specialistUnitListing/specialistUnitListing.schema";
+import SpecialistUnitListing from "@/components/TeacherViews/SpecialistUnitListing/SpecialistUnitListing.view";
+
+export type SpecialistUnitListingPageProps = {
+  curriculumData: SpecialistUnitListingData;
+};
+
+const SpecialistUnitListingPage: NextPage<SpecialistUnitListingPageProps> = ({
+  curriculumData,
+}) => {
+  const { subjectTitle, developmentalStageSlug, developmentalStage } =
+    curriculumData;
+
+  const developmentalStagesSEO = {
+    ...getSeoProps({
+      title: `Free Specialist ${subjectTitle} Teaching Resources for Lesson Planning`,
+      description: `We have resources for developmental stages: ${developmentalStage
+        .map((stage) => stage.title)
+        .join(", ")}`,
+    }),
+    ...{ noFollow: true, noIndex: true },
+  };
+
+  const unitsSEO = {
+    ...getSeoProps({
+      title: `Free Specialist ${subjectTitle} Teaching Resources for Lesson Planning`,
+      description: "Specialist programme units",
+    }),
+    ...{ noFollow: true, noIndex: true },
+  };
+
+  return (
+    <AppLayout
+      seoProps={developmentalStageSlug ? developmentalStagesSEO : unitsSEO}
+    >
+      <SpecialistUnitListing curriculumData={curriculumData} />
```

Comment:
I can't see where it's happening but the unit cards are linking to the non specialist lesson listing page (missing `/specialist/` from the url)